takwimu/urls.py

```diff
@@ -29,7 +28,7 @@
         r'^{}/$'.format(PROFILES_GEOGRAPHY_REGEX),
         IndicatorsGeographyDetailView.as_view(),
         name='geography_detail'),
-    url(r'^profiles/$', ProfileView.as_view(), name='profiles'),
+    url(r'^profiles/(?P<country_slug>[\w-]+)', ProfileView.as_view(), name='profiles'),
```

Comment:
Why are we adding this? Wagtail already handles profiles pages through `hurumap_dashboard_urlpatterns`.

Reply:
I could not see any way to get the information of current selected page.

Removing this `url(r'^profiles/$', ProfileView.as_view(), name='profiles'),` still worked and I could not find where it was being handled, maybe you could help point me to the code, I could not find it.

Reply:
I'd start here @karimkawambwa for `ProfilePage`: https://github.com/TakwimuAfrica/Takwimu/blob/65c0d67172b1fd33b8f85c978c3851cd09bbee49/takwimu/models/dashboard.py#L500

You'd then do the same for `ProfileSectionPage`.

Reply:
This is cool, 
We should make it clear if we are going to move forward this way.
We handle pages in wagtail and I think it could be worth splitting up the file.